lib/Model/Event.php

```diff
@@ -40,16 +40,16 @@ final class Event implements LoadMetadataInterface
     /** @var EventLocation|null */
     private $location;
 
-    /** @var EventSponsors */
+    /** @var EventSponsors<EventSponsor> */
```

Comment:
This looks a bit weird… I mean you can't have `EventSponsors<SomeRandomClass>`, can you?

Reply:
phpstan was complaining about that elements have to be `EventSponsor`. It doesn't do it anymore so I guess one of my fixes that came after might cover those ones.